### DISCUSSION TITLE
How  can I plot the other axes (coronal and saggital) ??

### DISCUSSION THREAD

**Discussion by AgustinSaturni** [Project-MONAI/tutorials#1766] [General]
Hi everyone, I trained the spleen segmentation model (https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb) and now I'm trying to plot the results of my predictions applied in news tomographies. But the example only shows how to plot the axial view. How can I plot another view?? I tried with this:
        original_image = loader(test_output[0].meta["filename_or_obj"])
        plt.figure("check", (18, 6))
        plt.subplot(1, 2, 1)
        plt.imshow(original_image[:, 200,:], cmap="gray")
        plt.subplot(1, 2, 2)
        plt.imshow(test_output[0].detach().cpu()[1, :, 200, :])
        plt.show()

But the image that plots its wrong:

![image](https://github.com/user-attachments/assets/7d8015f9-702e-49c8-8aab-6257dc03fcde)

Thank you!

**Comment by KumoLiu** [comment]
Hi @AgustinSaturni, you can try `aspect='auto'` in the `plt.imshow`. It's just because of the proportions.
